### DISCUSSION TITLE
ImportMap: require for tabler is downloading older version

### DISCUSSION THREAD

**Discussion by avtarsm** [symfony/symfony#53498] [Q&A] [answered]
I am trying to install tabler via importmap and it is downloading the 1.0.0-alpha8 instead of the current 1.0.0-bet20.

Do let me know how to specify the version while downloading a package

**Comment by MatTheCat** [comment, marked as answer]
See https://github.com/tabler/tabler/issues/707#issuecomment-727611208:

> `tabler` package has been moved to `@tabler/core`

**Reply by avtarsm** [reply]
Thank you for guiding me to the tabler. I have run the below command to get the latest version

`symfony console importmap:require @tabler/core@beta`
